README.md

```diff
@@ -1 +1,43 @@
-# First-Portfolio
\ No newline at end of file
+# First Portfolio Website
+
+# Built with
+
+1. HTML
+2. CSS-Flexbox
+3. Gitlab
+4. Github
+5. VsCode editor
+
+# Getting started
+
+To set up the portfolio;
```

Comment:
- [OPTIONAL] kindly add all processes for getting started with the project locally. please add the whole process to start the project locally. imagine that someone uninitiated in programming visits your Project and he wants to visualize it in his machine. explain to him how he can do it in this part.